### ISSUE TITLE
Publish to Amazon (for Kindle Fire)

### ISSUE THREAD

**Issue by Offbeatmammal** [freezy/android-xbmcremote#75]
I know I could build from source, or install to a regular Android device, back up the .apk and sideload it to a kindle but hey... I'm lazy. the new ICS based Kindles are great little tablets and (especially if any of this http://forum.xbmc.org/showthread.php?tid=143242 comes true) I'd like to use it as the remote for my Raspberry Pi...

Kindle appstore is free to submit, and if it's a standard Andsroid app should be pretty much ready to go [https://developer.amazon.com/welcome.html] - if there's a ready built .apk somewhere I can grab and sideload happy to test it out and report back...

**Comment by freezy** [comment]
Kindle store is the same as the Amazon Appstore, right?

**Comment by Offbeatmammal** [comment]
yup, though you'd need to set up a (free) developer account same .apk should work fine as long as there are no Google Mobile Services dependencies. If there's a link to an apk I can smoke-test easily
